scripts/tests/cirque_tests.sh

```diff
@@ -37,6 +37,7 @@ CIRQUE_TESTS=(
     "EchoTest"
     "MobileDeviceTest"
     "InteractionModelTest"
+    "BasicClusterAttributes"
```

Comment:
Could you add some description on this test, per offline message, I think this is used to validate cluster between border router and end device?

Reply:
never mind, per offline chat, this would not cover border router test. thanks